src/rust/engine/fs/src/directory.rs

```diff
@@ -359,30 +359,29 @@ impl From<DigestTrie> for DirectoryDigest {
 impl DigestTrie {
   /// Create a DigestTrie from unique PathStats. Fails for duplicate items.
   pub fn from_path_stats(
```

Comment:
Should possibly change the name and docstring here.

Reply:
hehe